app/lib/main.dart

```diff
@@ -146,243 +161,251 @@ class ChatOverview extends StatelessWidget {
+                      child: Column(
+                        children: [
                           Row(
                             mainAxisAlignment: MainAxisAlignment.spaceAround,
-                            children: <Widget>[
+                            children: const <Widget>[
                               Icon(Icons.settings),
                             ],
                           ),
-                          Divider(),
-                          Text("Effektio 0.0.1"),
-                        ]))),
+                          const Divider(),
+                          const Text("Effektio 0.0.1"),
+                        ],
+                      ),
+                    ),
                   ),
                 ),
-              ],
-            ),
-          ),
-          bottomNavigationBar: BottomNavigationBar(
-            type: BottomNavigationBarType.fixed,
-            currentIndex: _currentIndex,
-            backgroundColor: Color(0xFF6200EE),
-            selectedItemColor: Colors.white,
-            unselectedItemColor: Colors.white.withOpacity(.6),
-            selectedFontSize: 0,
-            unselectedFontSize: 0,
-            onTap: (value) {
-              // Respond to item press.
-              setState(() => _currentIndex = value);
-            },
-            items: [
-              BottomNavigationBarItem(
-                label: 'News',
-                icon: Icon(Icons.dashboard),
-              ),
-              BottomNavigationBarItem(
-                label: 'FAQ',
-                icon: Icon(Icons.help_outline),
-              ),
-              BottomNavigationBarItem(
-                label: 'Community',
-                icon: Icon(Icons.groups),
-              ),
-              BottomNavigationBarItem(
-                label: 'Chat',
-                icon: Icon(Icons.chat),
               ),
             ],
           ),
-        ));
+        ),
+        bottomNavigationBar: BottomNavigationBar(
+          type: BottomNavigationBarType.fixed,
+          currentIndex: _tabIndex,
+          backgroundColor: const Color(0xFF6200EE),
+          selectedItemColor: Colors.white,
+          unselectedItemColor: Colors.white.withOpacity(.6),
+          selectedFontSize: 0,
+          unselectedFontSize: 0,
+          onTap: (value) {
+            // Respond to item press.
+            setState(() => _tabIndex = value);
+          },
+          items: const [
+            BottomNavigationBarItem(
+              label: 'News',
+              icon: Icon(Icons.dashboard),
+            ),
+            BottomNavigationBarItem(
+              label: 'FAQ',
+              icon: Icon(Icons.help_outline),
+            ),
+            BottomNavigationBarItem(
+              label: 'Community',
+              icon: Icon(Icons.groups),
+            ),
+            BottomNavigationBarItem(
+              label: 'Chat',
+              icon: Icon(Icons.chat),
+            ),
+          ],
+        ),
+      ),
+    );
```

Comment:
I take it all this is primarily re-formatting? Do you mind adding a `.editorconfig`-file for the dart-files in the root of the repo we adhere to? Otherwise this will become very annoying soon.

and/or dart/flutter formatting rules to CI ... if such a thing exists...

Reply:
Formatting and member count in the chat tab.